src/panels/config/devices/ha-config-device-page.ts

```
@@ -665,269 +661,235 @@ export class HaConfigDevicePage extends LitElement {
+            ? html`<div class="header-name">
+                <a href="/config/areas/area/${area.area_id}"
+                  >${this.hass.localize(
+                    "ui.panel.config.integrations.config_entry.area",
+                    { area: area.name || "Unnamed Area" }
+                  )}</a
+                >
+              </div>`
+            : ""}
+          <div class="header-right">
+            ${battery &&
+            (batteryDomain === "binary_sensor" || !isNaN(battery.state as any))
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)